linux/step04_all_omero_install.sh

```diff
@@ -6,13 +6,6 @@ set -eux
 
 . `dirname $0`/settings.env
 
-#start-install-omero-py
-# Install omero-py
-$VENV_SERVER/bin/pip install "omero-py>=5.8.0"
```

Comment:
I assume this cleanup is motivated by the fact `omero-py` is pulled as a server dependency via `pip install omero-server[default]` (the title of the PR is slightly frightening)?

Reply:
it is.
The title got picked up from commit message (partial). Updating